src/lz4.c

```diff
@@ -198,10 +198,13 @@ void  LZ4_free(void* p);
 # define ALLOC_AND_ZERO(s) LZ4_calloc(1,s)
 # define FREEMEM(p)        LZ4_free(p)
 #else
-# include <stdlib.h>   /* malloc, calloc, free */
-# define ALLOC(s)          malloc(s)
-# define ALLOC_AND_ZERO(s) calloc(1,s)
-# define FREEMEM(p)        free(p)
+struct rdkafka_s;
```

Comment:
Why is this different from lz4frame.c ?

Reply:
I don't exactly remember, but I think in this case we couldn't include "rd.h"

Reply:
It'd be good if we could use rd.h here

Reply:
lz4.h is included only internally in the lz4 code, and I think some flags differ. Including either rd.h or rdkafka_int.h will open a huge can of worms (compiler errors)